api/app/controllers/api/v1/applicants_controller.rb

```diff
@@ -16,6 +16,11 @@ def index
 
         # POST /applicants
         def create
+            # if we passed in an id that exists, we want to update
```

Comment:
To clarify this defines two routes:
`/applicants` which will create an applicant based on the things passed in.
`/applicants/:id` which will update an existing applicant based on ID.

Side note: if you pass in an `id` that does not exist, this will behave like `/applicants` and ignore the id parameter. Is this intended or should we give a `id not found` error?

Reply:
I don't know if the former or latter is the intended behaviour. Usually, I would think it is the latter, which will make this implementation. However, when I was writing this section, @siefkenj said I should follow the format he used for this `update` vs. `create` separation. You would have to ask him for more details.

Reply:
I ended up creating `/applicants/:id` sort of `update` routes instead of rerouting via `create`.